README.md

```diff
@@ -231,21 +327,9 @@ Finally run both performance testing script & danger in your CI config:
 
 You can also check our example [GitHub workflow](https://github.com/callstack/reassure/blob/main/.github/workflows/main.yml).
 
-> **Note**: Your performance test will run much longer than regular integration tests. It's because we run each test scenario multiple times (by default 10), and we repeat that for two branches of your code. Hence, each test will run 20 times by default. That's unless you increase that number even higher.
-
-### Optional: ESLint setup
+The above example is based on GitHub Actions, but it should be similar to other CI config files and should only serve as a reference in such cases.
 
-ESLint might require you to have at least one `expect` statement in each of your tests. In order to avoid this requirement
-for performance tests you can add following override to your `.eslintrc` file:
-
-```js
-rules: {
- 'jest/expect-expect': [
- 'error',
-    { assertFunctionNames: ['expect', 'measurePerformance'] },
-  ],
-}
-```
+> **Note**: Your performance test will run much longer than regular integration tests. It's because we run each test scenario multiple times (by default 10), and we repeat that for two branches of your code. Hence, each test will run 20 times by default. That's unless you increase that number even higher.
```

Comment:
```suggestion
> **Note**: Your performance test will run significantly longer than regular integration tests. It's because we run each test scenario multiple times (by default 10), and we repeat that for two branches of your code. Hence, each test will run 20 times by default. That's unless you increase that number even higher.
```